src/org/parosproxy/paros/Constant.java

```diff
@@ -1264,6 +1266,17 @@ public static void setDevMode(boolean devMode) {
         Constant.devMode = devMode;
     }
     
+    /**
+     * Sets whether or not ZAP should be 'silent' ie not make any unsolicited requests.
+     * <p>
+     * <strong>Note:</strong> This method should be called only by bootstrap classes.
+     * 
+     * @param devMode {@code true} if ZAP should be silent, {@code false} otherwise.

```

Comment:
- Parameter name needs to be updated.
 - Could have since tags (same for getter).
 - Needs ZAP comment.